legacy_promotions/spec/rails_helper.rb

```diff
@@ -72,8 +95,18 @@
     Rails.cache.clear
   end
 
-  config.include Spree::TestingSupport::JobHelpers
+  config.define_derived_metadata(file_path: %r{spec/features/solidus_admin}) do |metadata|
+    metadata[:solidus_admin] = true
```

Comment:
Not sure to follow here. Can't we just set this metadata on the specs we want to test with `DummyApp.use_solidus_admin = true` as we are doing for the `flaky` metatag, for example?

Reply:
Ah ok, so the goal was to infer it from the folder without changing the specs? I think it might work but at the end we can add it to the first definition on top of the file and it will be used by all of them. Also, we might want to use it in specific specs into some other files that do not belong to that directory, and it might be a good idea to keep consistency and less magic (for things like search and replace, or similar). If you like this solution, though, I'm not opposed so I'm going to approve, up to you!